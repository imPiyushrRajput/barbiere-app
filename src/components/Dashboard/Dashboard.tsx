import { PageContent } from '@components/layout';
import { Text } from '@medly-components/core';
import React from 'react';

export const Dashboard: React.SFC = () => (
    <PageContent>
        <Text textWeight="Strong" textVariant="body1">
            Dashboard Content
        </Text>
    </PageContent>
);
