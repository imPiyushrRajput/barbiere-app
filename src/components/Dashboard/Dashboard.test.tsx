import { render } from '@test-utils';
import React from 'react';
import { Dashboard } from './Dashboard';

describe('Dashboard', () => {
    it('should render properly', () => {
        const { container } = render(<Dashboard />);
        expect(container).toMatchSnapshot();
    });
});
