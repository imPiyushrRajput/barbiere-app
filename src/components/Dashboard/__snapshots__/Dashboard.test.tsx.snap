// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should render properly 1`] = `
.c1 {
  margin: 0;
  color: inherit;
  font-size: 1.6rem;
  font-weight: 700;
  -webkit-letter-spacing: 0rem;
  -moz-letter-spacing: 0rem;
  -ms-letter-spacing: 0rem;
  letter-spacing: 0rem;
  line-height: 2.6rem;
}

.c0 {
  background-color: #eff2f4;
  overflow: auto;
  padding-top: 32px;
}

@media (max-width:1279px) {
  .c0 {
    padding-left: 24px;
    padding-right: 24px;
  }
}

@media (min-width:1280px) {
  .c0 {
    padding-left: calc(24px + (((100vw - 1280px) / 160) * 16));
    padding-right: calc(24px + (((100vw - 1280px) / 160) * 16));
  }
}

@media (min-width:1440px) {
  .c0 {
    padding-left: 40px;
    padding-right: 40px;
  }
}

<div>
  <main
    class="c0"
  >
    <strong
      class="c1"
    >
      Dashboard Content
    </strong>
  </main>
</div>
`;
